config/dev.exs

```diff
@@ -104,7 +104,7 @@ config :logflare, Logflare.Vercel.Client,
   install_vercel_uri: "https://vercel.com/integrations/logflare-v2-dev/new"
 
 config :goth,
-  json: "config/secrets/logflare-dev-238720-63d50e3c9cc8.json" |> File.read!()
+  json: "config/secrets/logflare-dev-238720-344624644919.json" |> File.read!()
```

Comment:
@chasers if we're using different dev jwt secrets now, we shld move this to `dev.secrets.exs`. I'm still using `63d50e3c9cc8`

Reply:
Agreed, I can tackle that in a fix-it-up PR.

We should explore ways of making this a bit easier for development (e.g. mock authentications for development?)

Reply:
Yeah this is fine.  This new one is in Bitwarden btw.

Reply:
will move to dev.secrets.exs